packages/kit/src/utils/error.spec.js

```diff
@@ -0,0 +1,17 @@
+import { test } from 'uvu';
+import * as assert from 'uvu/assert';
+import { coalesce_to_error } from './error.js';
+
+test('coalesce_to_error', () => {
```

Comment:
this is at least a little more complicated than the last test, but still would think this functionality is likely already covered by an integration test and that there's little room to go wrong here. I and the other maintainers agree that we should rely more on unit tests than integration tests, but I think the units could be for more complicated functionality

Reply:
test removed